source/server/overload_manager_impl.cc

```diff
@@ -72,15 +114,21 @@ Stats::Gauge& makeGauge(Stats::Scope& scope, absl::string_view a, absl::string_v
 
 OverloadAction::OverloadAction(const envoy::config::overload::v3::OverloadAction& config,
                                Stats::Scope& stats_scope)
-    : active_gauge_(
-          makeGauge(stats_scope, config.name(), "active", Stats::Gauge::ImportMode::Accumulate)) {
+    : state_(OverloadActionState::inactive()),
+      active_gauge_(
+          makeGauge(stats_scope, config.name(), "active", Stats::Gauge::ImportMode::Accumulate)),
+      scale_value_gauge_(makeGauge(stats_scope, config.name(), "scale_value",
```

Comment:
Please add docs for this new stat (https://github.com/envoyproxy/envoy/blob/master/docs/root/configuration/operations/overload_manager/overload_manager.rst#statistics)
And also unit tests

Reply:
I think this is done as of ecad37b, please let me know if I'm missing something.

Reply:
Yup - I missed that. Thanks!